### ISSUE TITLE
data_dir not used by flighpath save

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#337]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/337) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We have introduced data_dir parameter by 1.7.0.

Somehow not all of our store/load methods use it as initial path

This needs a review.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

```
#!python
elif pickertype in ["qt", "default"]:
     filename = get_open_filename_qt(parent, title, dirname, filt)
```
looks like did not understand dirname

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

data_dir becomes used by the qt filepicker too, fixes #337

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss/stable (pull request #557)

data_dir becomes used by the qt filepicker too, fixes #337
